### PR TITLE
docs: Some correction in Documentation at install devtron/ configuration / ingress setup

### PR DESCRIPTION
**Pull request by jatin-jangir**
URL at these page are not correctly redirecting to their destination.
https://docs.devtron.ai/configurations-overview 

Intention of yaml file 
metadata:
  annotations: nginx.ingress.kubernetes.io/app-root: /dashboard

# Description

Fixes #

## How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Checklist:

* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR requires documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code.
* [ ] I have commented my code, particularly in hard-to-understand areas.
* [ ] I have tested it for all user roles.
* [ ] I have added all the required unit/api test cases.

## Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: docs/setup/configurations/configurations-overview.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ The configuration allows you to quickly roll back a configuration change if nece
 
 There are two ways you can perform configurations while setting up Devtron dashboard:
 
-* [Installation Configurations](setup/install/installation-configuration.md)
-* [Override Configurations](setup/install/override-default-devtron-installation-configs.md)
+* [Installation Configurations](https://github.com/devtron-labs/devtron/blob/main/docs/setup/install/installation-configuration.md)
```
Comment (SNe789) [inline docs/setup/configurations/configurations-overview.md:8]: The link will still be broken as its coming from Github (after merging to main). Can you please add the link from the document itself? It is: https://docs.devtron.ai/configurations-overview/installation-configuration

File: docs/setup/configurations/configurations-overview.md [hunk 2]
```diff
@@ -5,8 +5,8 @@ The configuration allows you to quickly roll back a configuration change if nece
...
+* [Override Configurations](https://github.com/devtron-labs/devtron/blob/main/docs/setup/install/override-default-devtron-installation-configs.md)
```
Comment (SNe789) [inline docs/setup/configurations/configurations-overview.md:9]: Same here also: https://docs.devtron.ai/configurations-overview/override-default-devtron-installation-configs

File: docs/setup/configurations/configurations-overview.md [hunk 3]
```diff
@@ -5,8 +5,8 @@ The configuration allows you to quickly roll back a configuration change if nece
...
 
 
-You can also setup `ingress` while setting up Devtron dashboard. Refer [here](setup/install/ingress-setup.md) for ingress setup.
\ No newline at end of file
+You can also setup `ingress` while setting up Devtron dashboard. Refer [here](https://github.com/devtron-labs/devtron/blob/main/docs/setup/install/ingress-setup.md) for ingress setup.
```
Comment (SNe789) [inline docs/setup/configurations/configurations-overview.md:12]: Same here also: https://docs.devtron.ai/configurations-overview/ingress-setup

File: docs/setup/install/ingress-setup.md [hunk 4]
```diff
@@ -14,7 +14,8 @@ You can use [this yaml file](https://github.com/devtron-labs/devtron/blob/main/m
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  annotations: nginx.ingress.kubernetes.io/app-root: /dashboard
+  annotations: 
+    nginx.ingress.kubernetes.io/app-root: /dashboard
```
Comment (SNe789) [inline docs/setup/install/ingress-setup.md:18]: what is the difference between the previous `annotations` and the one you corrected?

Comment (jatin-jangir) [reply docs/setup/install/ingress-setup.md:18]: the previous one will create error regarding indentation in YAML file.
putting them in same line cause error.

**Review by prakarsh-dt — APPROVED** [review]

**Pull request merged**